client/src/main/java/com/chattycathy/client/handler/ClientStompSessionHandler.java

```diff
@@ -33,26 +37,49 @@ public ClientStompSessionHandler(Scanner scanner) {
     /**
      * A custom implementation that runs after connecting that subscribes to the needed events
      * and handles message sending
+     * Also checks if user message is a command, handling it if so.
      */
     @Override
     public void afterConnected(StompSession session, StompHeaders connectedHeaders) {
         log.info("Connected to Chatty Cathy");
         log.debug("Connected successfully to session {}, headers: {}", session, connectedHeaders);
+        log.info("Type '/help' to See The List of Available Commands");
         log.info("Please type to chat!");
 
         session.subscribe("/topic/main", this);
 
         new Thread(() -> {
+
+            List<Command> commandList = getComandList();
+            CommandHandler commandHandler = new CommandHandler(commandList);
+            String commandReturn;
+
             while (session.isConnected()) {
                 Message message = new Message(user.getUserName(), scanner.nextLine());
-                if (!message.getMessage().isEmpty()) {
+                commandReturn = commandHandler.runCommand(message.getMessage());
+
+                if (!message.getMessage().isEmpty() && commandReturn == null) {
```

Comment:
this if block can be simplified
```suggestion
                if (commandReturn != null) {
                    log.info(commandReturn);
                }
                else if (!message.getMessage().isEmpty()) {
                    session.send("/app/main", message);
                }
```

Reply:
i agree, pushed this change